static/src/js/util/url/collectionSortKeyEncoders.js

```diff
@@ -0,0 +1,21 @@
+import { getApplicationConfig } from '../../../../../sharedUtils/config'
+
+const { collectionSearchResultsSortKey: defaultSortKey } = getApplicationConfig()
+
+export const encodeCollectionSortKey = (collectionSortKey, collectionSortPreference) => {
```

Comment:
This is hiding the value from the url if it matches either the default or the preference. if my preference is not the default and it matches the default, it should be encoded

Reply:
I'm still working on trying to get the page load working correctly with the user preference